src/stylesheets/elements/_list.scss

```diff
@@ -1,20 +1,14 @@
 ul,
 ol {
-  @include margin(2em null 2em 0.9em);
-  list-style: none;
-  padding-left: 0;
-  display: table;
+  padding-left: 1.94em; // Approximately 15px left padding at default font size
```

Comment:
To keep it relative to the font size of the list: a bigger font = bigger padding.